SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -212,8 +212,9 @@ private void checkDocker(SingularityDeploy deploy) {
       final int numPorts = deploy.getResources().get().getNumPorts();
 
       if (!dockerInfo.getPortMappings().isEmpty()) {
-        checkBadRequest(dockerInfo.getNetwork().or(Protos.ContainerInfo.DockerInfo.Network.HOST) == Protos.ContainerInfo.DockerInfo.Network.BRIDGE,
-            "Docker networking type must be BRIDGE if port mappings are set");
+        checkBadRequest(dockerInfo.getNetwork().or(Protos.ContainerInfo.DockerInfo.Network.HOST) == Protos.ContainerInfo.DockerInfo.Network.BRIDGE ||
+            dockerInfo.getNetwork().or(Protos.ContainerInfo.DockerInfo.Network.HOST) == Protos.ContainerInfo.DockerInfo.Network.HOST,
```

Comment:
Ah, you are correct on that,  on closer read I misinterpreted the documentation, I'll fix this